lib/livebook/teams/org.ex

```diff
@@ -41,5 +41,6 @@ defmodule Livebook.Teams.Org do
     org
     |> cast(attrs, @fields)
     |> validate_required(@required_fields)
+    |> validate_format(:name, ~r/^[a-z0-9]+(?:-[a-z0-9]+)*$/)
```

Comment:
Would this work? And let's remember to validate in the server too (not sure if you haven't done that yet)!

```suggestion
    |> validate_format(:name, ~r/^[a-z0-9][a-z0-9\w]*$/)
```